host/host.cpp

```diff
@@ -55,7 +55,7 @@ class RPCHandler : virtual public RPCIf {
 
   void access(std::string& _return, const Operation& operation) {
     std::string key = operation.key;
-    std::string val = rd.get("1");
+    std::string val = rd.get(key);
     std::string update_val = operation.value;
```

Comment:
Inconsistently accessing things from operation. You alias `key` but don't alias `value`. Stick to one of them. My preference is `operation.key` and `operation.value` personally. I don't see the need to alias

Reply:
I think I aliased operation.value as update_val. I see your point though, I’ll remove the aliases